Configs/JumpCloud.SDK.DirectoryInsights.yaml

```diff
@@ -33,7 +33,7 @@ input-file:
 module-name: JumpCloud.SDK.DirectoryInsights
 namespace: JumpCloud.SDK.DirectoryInsights
 sample-generation: true
-module-version: 0.0.33
+module-version: 0.0.34
```

Comment:
Shouldn't we only be updating V1 here?

Reply:
These are previous changes from Geoff branch, not releasing in this PR/ not worried about it currently